build_tools/azure/install.sh

```diff
@@ -43,7 +43,7 @@ if [[ "$DISTRIB" == "conda" ]]; then
     fi
 
     TO_INSTALL="python=$PINNED_PYTHON_VERSION pip pytest=$PYTEST_VERSION \
-                pytest-cov numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION \
+                pytest-cov numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION psutil \
```

Comment:
Adding this to all test runs implies that it is a required dependency. We are better off adding this to one test run and skipping the test on other platforms.